sdl-core/src/main/scala/io/smartdatalake/workflow/dataobject/Table.scala

```diff
@@ -30,7 +30,23 @@ import org.apache.spark.sql.catalyst.TableIdentifier
  *           Called db for backwards-compatibility because for hive tables, db and schema mean the same thing.
  * @param name        table name
  * @param query       optional select query
+ * @param commentOnTable  An optional comment to add to the table after writing a DataFrame to it. As of now, this is only possible for Delta Lake and Snowflake tables.
```

Comment:
We should not create another attribute for this, but use DataObject.metdata.description.